src/main/java/com/avispl/symphony/dal/avdevices/encoderdecoder/haivision/x4encoder/HaivisionX4EncoderCommunicator.java

```diff
@@ -803,21 +944,787 @@ else if (EncodingProfile.MAIN_10.getName().equals(encodingProfileInput) || Encod
 				String[] dropdownReSyncHour = DropdownList.Names(ReSyncHourDropdown.class);
 				AdvancedControllableProperty reSyncHourControlProperty = controlDropdown(extendedStatistics, dropdownReSyncHour, property, value);
 				addAdvanceControlProperties(advancedControllableProperties, reSyncHourControlProperty);
+
+				videoResponseDailyReSync = videoNameToVideoResponse.get(videoName);
+				videoResponseDailyReSync.setReSyncHour(value);
 				break;
 			default:
```

Comment:
can you please put log entries into the default state, just to keep track of what's going on, if needed